docs/extending.md

```diff
@@ -0,0 +1,71 @@
+# Extending nf-core/multiplesequencealign
```

Comment:
Isn't this covered by the Contributing Guidelines in `.github/CONTRIBUTING.md`?

Reply:
Not completely - here it specifies precisely how to do it for this specific pipeline (which parts of the workflow to touch etc). I added now a pointer to CONTRIBUTING from the extending